vowpalwabbit/constant.h

```diff
@@ -18,6 +18,7 @@ constexpr float probability_tolerance = 1e-5f;
 constexpr uint32_t FNV_prime = 16777619;
 
 constexpr unsigned char default_namespace = 32;
+constexpr unsigned char wildcard_namespace = 58;  // :
```

Comment:
nit: It'd be nice to namespace these constants at some point...

Reply:
I agree, but a TODO for another day